.travis.yml

```diff
@@ -3,24 +3,29 @@ language: python
 matrix:
   include:
     - python: "2.7"
-      env: DJANGO=1.8 COVERAGE=YES
+      env: DJANGO=1.11 COVERAGE=YES
+    - python: "3.4"
+      env: DJANGO=1.11
     - python: "3.5"
-      env: DJANGO=1.8
+      env: DJANGO=1.11
+    - python: "3.6"
+      env: DJANGO=1.11 
 
-    - python: "2.7"
-      env: DJANGO=1.9
+    - python: "3.4"
+      env: DJANGO=2.0 
     - python: "3.5"
-      env: DJANGO=1.9 COVERAGE=YES
+      env: DJANGO=2.0
+    - python: "3.6"
+      env: DJANGO=2.0 COVERAGE=YES FLAKE8=YES
 
-    - python: "3.4"
-      env: FLAKE8=YES
+before_install:
+  - sudo apt-get -qq update
+  - sudo apt-get install -y libxml2-utils
```

Comment:
Does this PR now depend on `libxml2-utils` being available? Or only for testing?

Reply:
This is needed since you changed the way to validate the xml, see commit: 69dc92c5cb557a7bd09b9a9316d93d76c6d2e390

Reply:
So yes, only for testing. It's not used by airbrake itself.